package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/starbase",
-  "version": "0.4.3",
+  "version": "0.5.0",
```

Comment:
We actually use `auto` for versioning, so you can back this change out (and any tag creations too).  I added the `minor` and `release` labels to the PR and it should handle all the necessary updates from there.

Reply:
Agree with your logic on the version number though.  I'd forgotten we were still at a pre-1.0 release for the project to be honest. :)